stdlib/gc.mli

```diff
@@ -45,10 +45,23 @@ type stat =
 
     live_words : int;
     (** Number of words of live data in the major heap, including the header
-       words. *)
+       words.
+
+       Note that "live" words refers to every word in the major heap that isn't
+       currently known to be collectable, which includes words that haven't been
+       reachable by the program since the start of the previous gc cycle. It is
+       typically much simpler and more predictable to call {!Gc.full_major} (or
+       {!Gc.compact}) then computing gc stats, as then "live" words has the
+       simple meaning of "reachable by the program". One caveat is that a single
+       call to {!Gc.full_major} will not reclaim values that have a finaliser
+       from {!Gc.finalise} (this does not apply to {!Gc.finalise_last}). If this
+       caveat matters, simply call {!Gc.finalise} twice instead of once.
```

Comment:
I think you mean `Gc.full_major` rather than `Gc.finalise`.

Reply:
Oops, yes, fixed.